csharp/extractor/Semmle.Extraction/Entities/File.cs

```diff
@@ -10,93 +10,54 @@ public class File : CachedEntity<string>
         File(Context cx, string path)
             : base(cx, path)
         {
-            Path = path;
+            OriginalPath = path;
+            TransformedPath = Context.Extractor.PathTransformer.Transform(OriginalPath);
```

Comment:
Looks like we need to guard against the empty string here (as passed from `GeneratedLocation`).